### ISSUE TITLE
Publish Release Version (non-snapshot)

### ISSUE THREAD

**Issue by A248** [KyoriPowered/adventure-text-feature-pagination#27]
Hello, I'm wondering if it's under consideration to publish a release version soon. Is there anything preventing a full release?

**Comment by zml2008** [comment]
Yes, this feature needs to be completely redesigned -- the current design is fairly limited, and a lot less user-friendly than, for example, what Sponge offers.
